ec2/spark_ec2.py

```diff
@@ -24,10 +24,12 @@
 import hashlib
 import logging
 import os
+import os.path
 import pipes
 import random
 import shutil
 import string
+from stat import s_IRUSR
```

Comment:
I believe there's a typo here. It should be: `S_IRUSR`

Also, do we need this import here if we're explicitly referencing `os.stat.S_IRUSR` in [R360](https://github.com/apache/spark/pull/4162/files#diff-ada66bbeb2f1327b508232ef6c3805a5R360)?